lib/cluster.js

```diff
@@ -694,8 +696,10 @@ function workerInit() {
     if (!this.isConnected()) {
       process.exit(0);
     } else {
-      send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-      process.once('disconnect', () => process.exit(0));
+      const processDisconnect = () => process.disconnect();
```

Comment:
I think my involvement in this is pretty much limited to my comments in https://github.com/nodejs/node/issues/8913 (which is probably the issue you have been looking for?).

There was some talk about this in a CTC meeting but it seems there is no recording of the meeting available, so the [minutes](https://docs.google.com/document/d/1dUPFNyK8Yn1yWRAcXPY-mWM2KMWvOOB8Z5BH2s7RpJs/edit) is all we have. (According to those I was present but tbh I don’t remember much about that part…)